substrate/client/consensus/common/src/import_queue.rs

```diff
@@ -97,7 +97,7 @@ pub struct IncomingBlock<B: BlockT> {
 
 /// Verify a justification of a block
 #[async_trait::async_trait]
-pub trait Verifier<B: BlockT>: Send + Sync {
+pub trait Verifier<B: BlockT, A = ()>: Send + Sync {
```

Comment:
Does it make sense to document what `A` type stands for? After reading the code once I don't remember how it is useful, and in general, looking at the trait and its generic types wouldn't be enough.